clients/appointments/src/components/ui.tsx

```diff
@@ -96,10 +76,12 @@ export const Ui = ({
               <Box mb="8px">
                 <H3>{dateString}</H3>
               </Box>
-              <Box mb="8px">
-                <H3>{locationTitle}</H3>
-                <a href={locationHref}>{locationAddress}</a>
-              </Box>
+              { location &&
+                <Box mb="8px">
+                  <H3>{location.title}</H3>
+                  <a href={location.href}>{location.address}</a>
+                </Box>
+              }
```

Comment:
I made this conditional because if we don't have the location config, these would all be `undefined`